Spatial_Engine/Query/Normal.cs

```diff
@@ -33,6 +35,9 @@ public static partial class Query
         /****            IElement2D            ****/
         /******************************************/
 
+        [Description("Returns the normal of the external outline curve.")]
+        [Input("element2D", "The IElement2D to get the normal from.")]
+        [Output("n", "The normal to the IElement2D which is perpendicular to its plane and oriented according to the rigth hand rule in relation to the outline curve.")]
```

Comment:
Really good description, but would maybe move some of this text from the output to the description of the method instead.